src/1_basics.test.js

```diff
@@ -4,16 +4,22 @@ import { multiply, isSmallerThan10 } from './1_basics';
 // https://www.youtube.com/watch?v=__QEPUdnJS0
 describe('1_Basics', () => {
   describe('multiply()', () => {
-    it('...', () => {
-      multiply(3, 5);
-      // your test here...
+    it('should return the product of two numbers', () => {
+      expect(multiply(3, 5)).toBe(15);
+      expect(multiply(0, 5)).toBe(0);
+      expect(multiply(-2, 4)).toBe(-8);
+     
     });
   });
 
   describe('isSmallerThan10()', () => {
-    it('Your tests here...', () => {
-      isSmallerThan10(5);
-      // your test here...
+    it('It should compare if the number is smaller than 10', () => {
```

Comment:
todo: no need to say "it should" again, saying "should" is enough because you already have the `it()` from the fn.

```suggestion
    it('should compare if the number is smaller than 10', () => {
```